src/darwin/Framework/CHIP/CHIPOnOff.h

```diff
@@ -19,16 +19,17 @@
 #define CHIP_ONOFF_H
```

Comment:
I have not included that in my example branch, but this file (and `CHIPOnOff.mm`) could be generated as well. 
I'm mostly saying it because in this case, depending on what is decided, it could be a single file for all clusters instead of one file per cluster.